test/test_train_imagenet.py

```diff
@@ -1,12 +1,23 @@
 import test_utils
 
+
+SUPPORTED_MODELS = ['resnet50']
```

Comment:
This needs to be a little bit different as, depending on the model, the various parameter default will change.
Unfortunately this is a chicken and egg problem since in order to override, we need the model.
What I'd do is to call `test_utils.parse_common_options()` with all the overrideables set as `None`, and then have an override function after that that looks up a dictionary based on model selection, and overrides the values is they are `None`.